### ISSUE TITLE
Documentation: Install on CentOS 6

### ISSUE THREAD

**Issue by savornicesei** [dotnet/coreclr#26315]
Hi all,

This is related to issue #6792 and [this](https://github.com/dotnet/core/blob/master/Documentation/build-and-install-rhel6-prerequisites.md) documentation page.

On a fresh CentOS 6.10, updating curl library breaks yum (it complains about not beeing able to read /etc/yum.conf file).
Setting LD_LIBRARY_PATH back to /usr/lib fixes yum.
But still, yum install fails because of the openssl-libs dependency that is not available on CentOS 6.
The only solution to bypass the yum dependencies check is to install the dotnet packages using rpm.

I have tried the following dotnet repositories:
* https://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
* https://packages.microsoft.com/config/rhel/6/packages-microsoft-prod.rpm
* https://packages.microsoft.com/config/centos/6/packages-microsoft-prod.rpm
but all the dotnet packages require the openssl-libs.

If it's possible to install dotnet from source on CentOS 6 then the documentation needs to be updated. But as it is now - it's not helping getting dotnet 2.x on CentOS 6.

Thank you,
Simo

**Comment by janvorli** [comment]
@savornicesei let me try to verify the steps from the doc (https://github.com/dotnet/core/blob/master/Documentation/build-and-install-rhel6-prerequisites.md). At the time I have written it, they were working on fresh CentOS 6 install. So maybe something has changed.
Btw, the LD_LIBRARY_PATH should not be set to /usr/local/lib permanently, only for running the .NET Core applications. The simplest way is to put LD_LIBRARY_PATH=/usr/local/lib at the beginning of the command line that starts the .NET core app. Then the curl built from sources cannot interfere with yum in any way.

**Comment by savornicesei** [comment]
@janvorli I have a VM with CentOS 6 that I can easily rollback to a clean system. Let me know if I can help in any way.

**Comment by janvorli** [comment]
@savornicesei I have just verified the steps provided in the document on a freshly created CentOS 6.10 docker container, which should be the most minimal version of CentOS 6.10. 
I have found three minor issues: 
* the dependency list contains krb5 package, which should be krb5-libs instead
* the command to build CURL from sources was missing backslash at the end of the line with `--prefix=$PWD/install/usr/local`.
* The `export LD_LIBRARY_PATH=/usr/local/lib` is not correct, you should never set the LD_LIBRARY_PATH this way as it is set for all the commands that you execute after that. It really needs to be set only for running .NET core apps (or the dotnet tool itself). Otherwise it breaks other apps like the yum.

But other than that, everything worked as expected. I've downloaded .NET Core SDK 2.2 after installing all the stuff and I was able to build and run the hello world app.

**Comment by janvorli** [comment]
Btw, I've downloaded the 2.2 sdk tarball from https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.2.401-linux-rhel-6-x64-binaries

**Comment by savornicesei** [comment]
Hi @janvorli 
Are you referring to [this](https://gist.github.com/janvorli/ea160cef429c88ff152c6923c3f0c1b9) docker container?
So it actually needs to be built from source not installed through yum. That makes more sense.

**Comment by janvorli** [comment]
No, I meant the vanilla centos:6.10 image. To make it clear, I've ran the following command to create and run the container in which I've then made all the steps described in the doc.
```bash
docker run -it centos:6.10 /bin/bash
```
